src/user-points/user-points.service.ts

```diff
@@ -11,6 +11,20 @@ import { EventType, Prisma, UserPoints } from '.prisma/client';
 export class UserPointsService {
   constructor(private readonly prisma: PrismaService) {}
 
+  async findOrThrow(userId: number): Promise<UserPoints> {
```

Comment:
Can we add a test, similar to the [`UsersService`](https://github.com/iron-fish/ironfish-api/blob/master/src/users/users.service.spec.ts#L61-L84)?

Reply:
done